core/simulations/helpers/createVaultWithCollateral.ts

```diff
@@ -21,11 +22,16 @@ import { CollateralConfig, CollateralType } from '../../src/types';
 import { roundDownToFirstSignificantDecimal, roundUpToFirstSignificantDecimal } from '../../helpers/hex';
 import { determineBalanceSlot, setCollateralInWallet } from '../../helpers/hardhat/erc20';
 import { getAllCollateralTypes } from '../../src/constants/COLLATERALS';
+import { grantAdminPrivelegeForContract } from '../../helpers/hardhat/slotOverwrite';
+import {
+    allowAllActionsInClipperContract,
+    setCollateralDebtCeilingToGlobal,
+    setCollateralLiquidationLimitToGlobal,
+} from '../../helpers/hardhat/contractParametrization';
+import { overwriteStabilityFeeAccumulationRate } from '../../helpers/hardhat/overwrites';
 
 const UNSUPPORTED_COLLATERAL_TYPES = [
     'CRVV1ETHSTETH-A', // Collateral handled differently
-    'UNIV2DAIUSDC-A', // Liquidation limit too high (fails with "Dog/liquidation-limit-hit")
-    'WSTETH-B', // Does not accumulate stability fee rate at all
     'RETH-A', // [temporary] this collateral is not yet deployed, tested via different flow
```

Comment:
The suggestion during some of our dailies was to remove `RETH-A` from here since we now (after it was deployed) should be able to test it as well using `file` methods. But we can also do it in another PR!